test/distributed/org/apache/cassandra/distributed/test/AuthTest.java

```diff
@@ -41,15 +56,53 @@ public void authSetupIsCalledAfterStartup() throws IOException
     {
         try (Cluster cluster = Cluster.build().withNodes(1).start())
         {
-            boolean setupCalled = cluster.get(1).callOnInstance(() -> {
-                long maxWait = TimeUnit.NANOSECONDS.convert(10, TimeUnit.SECONDS);
-                long start = System.nanoTime();
-                while (!StorageService.instance.authSetupCalled() && System.nanoTime() - start < maxWait)
-                    Uninterruptibles.sleepUninterruptibly(1, TimeUnit.SECONDS);
-
-                return StorageService.instance.authSetupCalled();
-            });
-            assertTrue(setupCalled);
+            IInvokableInstance instance = cluster.get(1);
+            await().pollDelay(1, SECONDS).pollInterval(1, SECONDS)
```

Comment:
this does not have any explicit "timeout", does it? what about adding `atMost`?

also, formatting would be nice to have like this:

````
await().pollDelay(1, SECONDS)
       .pollInterval(1, SECONDS)
       .atMost(...)
       .until(() -> instance.callOnInstance(() -> StorageService.instance.authSetupCalled()));
````